docset/windows/configci/New-CIPolicyRule.md

```diff
@@ -241,6 +264,23 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -FilePathRule
+Specifies the path of a file or folder path for generating a rule with level set to FilePath. 
```

Comment:
"path of a file or folder path" is weird. Why not just "path of a file or folder"

Reply:
Fixed the wording